apps/pie-docs/src/components/modal/code/events.json

```diff
@@ -0,0 +1,34 @@
+{
+  "headings": [
+    "Event",
+    "Type",
+    "Description"
+  ],
+  "rows": [
+    [
+      "`pie-modal-open`",
+      "`CustomEvent`",
+      "Triggered when the modal is opened."
+    ],
+    [
+      "`pie-modal-close`",
+      "`CustomEvent`",
+      "Triggered when the modal is closed."
+    ],
+    [
+      "`pie-modal-back `",
```

Comment:
Are these spaces intentional? Same on the next one too.

Reply:
noooo updated 🤝